koans/src/java7/AboutStringsInSwitch.java

```diff
@@ -0,0 +1,27 @@
+package java7;
+
+import com.sandwich.koan.Koan;
+
+import static com.sandwich.koan.constant.KoanConstants.__;
+import static com.sandwich.util.Assert.assertEquals;
+
+public class AboutStringsInSwitch {
+
+    @Koan
+    public void stringsInSwitchStatement () {
+        String[] animals = {"Dog", "Cat", "Tiger", "Elephant", "Zebra"};
+        String dangerous=null;
+        String notDangerous=null;
+        for (String animal : animals){
+            switch (animal) {
+                case "Tiger":
+                        dangerous = animal;
```

Comment:
I'd say it's mostly a matter of taste, but I'm also thinking the compiler wouldn't like the continue statement because its implied in case statements. See http://stackoverflow.com/questions/2521721/why-cant-i-use-continue-inside-a-switch-statement-in-java